### PR TITLE
Feature/people

### PR DESCRIPTION
**Pull request by larsyaeger**

### CODE REVIEW

File: app/(guarded)/people/SongCard.jsx [hunk 1]
```diff
@@ -0,0 +1,21 @@
...
+          fill
+          src="https://thumbs.dreamstime.com/z/dog-cat-above-white-banner-5900263.jpg"
+        />
+      </div>
+      <div className="flex flex-col">
+        <h1 className="mt-1 ml-[2vh] text-[2.3vh] cursor-default">Cat Song</h1>
+        <h1 className="mt-1 ml-[2vh] text-[2.3vh] cursor-default">Mittens</h1>
+        <h1 className="mt-1 ml-[2vh] text-[2.3vh] cursor-default">Mrrreowww</h1>
+      </div>
+    </div>
```
Comment (CorgiOnNeptune) [inline app/(guarded)/people/SongCard.jsx]: I think the album cover would look a little better with some more margin and maybe some more border-radius too. Heavier weight for the font too?

File: app/(guarded)/people/UserCard.jsx [hunk 2]
```diff
@@ -0,0 +1,21 @@
...
+          src="https://thumbs.dreamstime.com/b/cute-cat-portrait-square-photo-beautiful-white-closeup-105311158.jpg"
+        />
+      </div>
+      <h2 className="text-[1.8vh] text-center cursor-default">Elon Musk</h2>
+      <h2 className="text-[1.8vh] text-center cursor-default">5.6km away</h2>
+      <button type="submit">
+        <h2 className="text-[1.8vh]">Send DM</h2>
+      </button>
+    </div>
+  );
```
Comment (CorgiOnNeptune) [inline app/(guarded)/people/UserCard.jsx]: Just a couple personal style preference suggestions to try out I think would look nice. 
```suggestion
export default function UserCard() {
  return (
    <div className="flex flex-col justify-evenly max-w-[250px] min-w-[10vw] w-[15vw] m-5 bg-primary p-2 h-2/3 min-h-[20rem] rounded-lg ">
      <div className="sticky aspect-square max-w-full min-h-2/5 h-2/4 self-center my-3">
        <Image
          alt="cute cat"
          className="rounded-lg block m-auto aspect-square"
          fill
          src="https://thumbs.dreamstime.com/b/cute-cat-portrait-square-photo-beautiful-white-closeup-105311158.jpg"
        />
      </div>
      <h2 className="text-[1.8vh] text-center font-bold cursor-default">Elons' Musk</h2>
      <h2 className="text-[1.8vh] text-center text-secondary cursor-default">5.6km away</h2>
      <button type="submit">
        <h2 className="text-[1.8vh]">Send DM</h2>
      </button>
    </div>
  );
```

Comment (CorgiOnNeptune) [inline app/(guarded)/people/UserCard.jsx]: Also maybe use the button component for the Send DM?
```suggestion
import Image from 'next/image';
import Button from 'app/(button)/Button';
import { MESSAGES } from 'public/constants/pathNames';

export default function UserCard() {
  return (
    <div className="flex max-w-[250px] min-w-[10vw] w-[15vw] m-5 flex-col bg-primary p-2 h-2/3 min-h-[20rem] rounded-lg ">
      <div className="sticky aspect-square max-w-full min-h-2/5 h-2/4 self-center my-3">
        <Image
          alt="cute cat"
          className="rounded-lg block m-auto aspect-square"
          fill
          src="https://thumbs.dreamstime.com/b/cute-cat-portrait-square-photo-beautiful-white-closeup-105311158.jpg"
        />
      </div>
      <h2 className="text-[1.8vh] text-center cursor-default">Elon Musk</h2>
      <h2 className="text-[1.8vh] text-center cursor-default">5.6km away</h2>
      <div className="flex justify-center text-[1.8vh]">
        <Button addedclasses="text-sm" content="Send DM" path={MESSAGES} />
      </div>
    </div>
  );
```

File: app/(guarded)/people/page.jsx [hunk 3]
```diff
@@ -1,5 +1,43 @@
...
+          <h1 className="text-[1.5vw] p-5 cursor-default">x user/s found </h1>
+          <button
+            className="ml-[17vw]"
+            onClick={() => setShowMap(!showMap)}
+            type="submit">
+            <h2 className="text-[1.5vw]">View On Map</h2>
+          </button>
+        </div>
+        <div className="flex flex-row h-3/4 w-full overflow-x-auto overflow-y-clip items-center">
+          {showMap ? <Map /> : null}
```
Comment (CorgiOnNeptune) [inline app/(guarded)/people/page.jsx]: Could use some margin so the map isn't hugging the containers side
```suggestion
          <button
            className="ml-[19vw]"
            onClick={() => setShowMap(!showMap)}
            type="submit">
            <h2 className="text-[1.5vw]">View On Map</h2>
          </button>
        </div>
        <div className="flex flex-row h-3/4 w-full overflow-x-auto overflow-y-clip items-center m-4">
          {showMap ? <Map /> : null}
```

File: app/(guarded)/people/page.jsx [hunk 4]
```diff
@@ -1,5 +1,43 @@
...
+      </div>
+      <div className="container m-auto bg-secondary w-4/5 h-3/5 max-h-max rounded-3xl">
+        <div className="flex flex-row">
+          <h1 className="text-[1.5vw] p-5 cursor-default">x user/s found </h1>
+          <button
+            className="ml-[17vw]"
+            onClick={() => setShowMap(!showMap)}
+            type="submit">
+            <h2 className="text-[1.5vw]">View On Map</h2>
+          </button>
```
Comment (CorgiOnNeptune) [inline app/(guarded)/people/page.jsx]: Could probably also reuse the button component again here.

**Review by CorgiOnNeptune — CHANGES_REQUESTED** [review]
Looks great! Just a couple of suggestions/options to try on some styling and stuff.

File: app/(guarded)/people/page.jsx [hunk 5]
```diff
@@ -1,5 +1,43 @@
...
+        </div>
+        <div className="flex flex-row h-3/4 w-full overflow-x-auto overflow-y-clip items-center">
```
Comment (Likai-L) [inline app/(guarded)/people/page.jsx]: Could try out grid layout here if you have time

**Review by Likai-L — APPROVED** [review]

**Review by CorgiOnNeptune — APPROVED** [review]

**Pull request merged**